config/validate_customfee_test.go

```diff
@@ -0,0 +1,66 @@
+package config_test
+
+import (
+	"testing"
+
+	"github.com/bitcoin-sv/spv-wallet/config"
+	"github.com/stretchr/testify/require"
+)
+
+func TestValidateFeeUnit(t *testing.T) {
+	tests := map[string]struct {
+		customFee  *config.FeeUnitConfig
+		shouldFail bool
+	}{
+		"Not defined is valid": {
+			customFee: nil,
+		},
+		"Standard": {
+			customFee: &config.FeeUnitConfig{
+				Satoshis: 1,
+				Bytes:    1000,
+			},
+		},
+		"Zero Satoshi is valid": {
+			customFee: &config.FeeUnitConfig{
+				Satoshis: 0,
+				Bytes:    1000,
+			},
+		},
+		"Empty is not ok": {
+			customFee:  &config.FeeUnitConfig{},
+			shouldFail: true,
+		},
+		"Negative satoshis": {
+			customFee: &config.FeeUnitConfig{
+				Satoshis: -1,
+				Bytes:    1000,
+			},
+			shouldFail: true,
+		},
+		"Zero bytes": {
+			customFee: &config.FeeUnitConfig{
+				Satoshis: 1,
+				Bytes:    0,
+			},
+			shouldFail: true,
+		},
+		"Negative bytes": {
+			customFee: &config.FeeUnitConfig{
+				Satoshis: 1,
+				Bytes:    -1,
+			},
+			shouldFail: true,
+		},
+	}
+	for name, test := range tests {
+		t.Run(name, func(t *testing.T) {
+			err := test.customFee.Validate()
```

Comment:
Can you do it like int the other tests for validate in this package 🙈 
IMHO you can quite simply copy example from validate_paymail_test.go
Ohhh and please split success and error test cases.

Why?
Because they are testing the validate at the AppConfig level not at some level below, so we know that it will be validated when called at the whole app config.
It's easier to understand or change that many tests if they all looks very similar.

Reply:
Ok Ok 😄 
You've introduced some cool method how to do config tests and I didn't noticed that 🙈 

I'm adjusting :D